src/Compilers/CSharp/Test/IOperation/IOperation/FunctionPointerOperations.cs

```diff
@@ -0,0 +1,538 @@
+                  Operand: 
+                    IOperation:  (OperationKind.None, Type: System.Int32, IsInvalid) (Syntax: 'ptr(Prop)')
+                      Children(2):
+                          IParameterReferenceOperation: ptr (OperationKind.ParameterReference, Type: delegate*<System.String, System.Int32>, IsInvalid) (Syntax: 'ptr')
+                          IPropertyReferenceOperation: System.String C.Prop { get; } (OperationKind.PropertyReference, Type: System.String, IsInvalid) (Syntax: 'Prop')
+                            Instance Receiver: 
+                              IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: C, IsInvalid, IsImplicit) (Syntax: 'Prop')
+      Initializer: 
+        null
+  IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsInvalid) (Syntax: 's = ptr(Prop);')
+    Expression: 
+      ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.String, IsInvalid) (Syntax: 's = ptr(Prop)')
+        Left: 
+          ILocalReferenceOperation: s (OperationKind.LocalReference, Type: System.String) (Syntax: 's')
+        Right: 
+          IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: System.String, IsInvalid, IsImplicit) (Syntax: 'ptr(Prop)')
+            Conversion: CommonConversion (Exists: False, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+            Operand: 
+              IOperation:  (OperationKind.None, Type: System.Int32, IsInvalid) (Syntax: 'ptr(Prop)')
+                Children(2):
+                    IParameterReferenceOperation: ptr (OperationKind.ParameterReference, Type: delegate*<System.String, System.Int32>, IsInvalid) (Syntax: 'ptr')
+                    IPropertyReferenceOperation: System.String C.Prop { get; } (OperationKind.PropertyReference, Type: System.String, IsInvalid) (Syntax: 'Prop')
+                      Instance Receiver: 
+                        IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: C, IsInvalid, IsImplicit) (Syntax: 'Prop')
+";
+
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // (7,20): error CS0029: Cannot implicitly convert type 'int' to 'string'
+                //         string s = ptr(Prop);
+                Diagnostic(ErrorCode.ERR_NoImplicitConv, "ptr(Prop)").WithArguments("int", "string").WithLocation(7, 20),
+                // (8,13): error CS0029: Cannot implicitly convert type 'int' to 'string'
+                //         s = ptr(Prop);
+                Diagnostic(ErrorCode.ERR_NoImplicitConv, "ptr(Prop)").WithArguments("int", "string").WithLocation(8, 13)
+            };
+
+            VerifyOperationTreeAndDiagnosticsForTest<BlockSyntax>(comp, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void FunctionPointerLoad_InCFG()
```

Comment:
>Load [](start = 35, length = 4)

"Assignment"?

Reply:
No, the interesting thing I'm testing here is the address of. I'll use that in the name.

---
In reply to: [431389332](https://github.com/dotnet/roslyn/pull/44436#discussion_r431389332) [](ancestors = 431389332)